vagrant_setup.py

```diff
@@ -23,13 +23,13 @@ def get_usb_devices():
         product = find_line_items("Product", device)
 
         if (len(productid) < 1 or len(vendorid) < 1 or 
-                len(manufacturer) < 1 or len(product) < 1):
+                len(manufacturer) < 1):
```

Comment:
`VBoxManage list usbhost` on my Windows test machine says that the internal Bluetooth adapter doesn't have a Product name.